### PR TITLE
Adding file for server connection policy

### PR DESCRIPTION
**Pull request by AmolAgarwal**
This currently can be used to update server connection type.

### CODE REVIEW

File: arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/connectionPolicies/Default": {
+            "put": {
+                "tags": [
+                    "SecureServerConnection"
+                ],
+                "operationId": "Server_UpdateSecureConnectionPolicy",
+                "description": "Udpates Sever connection policy.",
```
Comment (akromm) [inline arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json]: Update*

File: arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ServerSecureConnectionPolicyProperties": {
+            "properties": {
+                "connectionType": {
+                    "type": "string",
```
Comment (akromm) [inline arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json]: If this requires a value from a specific set, please define an x-ms-enum with the valid values.  See: https://github.com/akromm/azure-rest-api-specs/blob/master/arm-sql/2014-04-01/swagger/sql.core.json#L1463

File: arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/connectionPolicies/Default": {
+            "put": {
+                "tags": [
+                    "SecureServerConnection"
+                ],
+                "operationId": "Server_UpdateSecureConnectionPolicy",
```
Comment (akromm) [inline arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json]: Server should be plural:
Servers_UpdateSecureConnectionPolicy

File: arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+                            "$ref": "#/definitions/ServerSecureConnectionPolicy"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "SecureConnection"
+                ],
+                "operationId": "Server_GetSecureConnectionPolicy",
```
Comment (akromm) [inline arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json]: Servers_GetSecureConnectionPolicy

File: arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json (hunk 3 above)
Comment (AmolAgarwal) [inline arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json]: It applies to only 1 server - do we still need it as plural.

File: arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json (hunk 2 above)
Comment (AmolAgarwal) [inline arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json]: fixed.

File: arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json (hunk 3 above)
Comment (akromm) [inline arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json]: Yes.  the "Server" part becomes a class name in the client library, so we don't want:
client.Server.Update...
client.Servers.CreateOrUpdate...

better to keep things consistent.

File: arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/connectionPolicies/Default": {
+            "put": {
+                "tags": [
+                    "SecureServerConnection"
+                ],
+                "operationId": "Servers_UpdateSecureConnectionPolicy",
+                "description": "Updates Sever connection policy.",
```
Comment (akromm) [inline arm-sql/2014-04-01/swagger/secureServerConnectionPolicy.json]: server*

**Comment by AmolAgarwal** [comment]
there are merge conflicts not resolving automatically. Will create a new one.

**Pull request closed without merging**